fiorella/src/sections/header.liquid

```diff
@@ -6,7 +6,95 @@
   {% endif %}
 </style>
 <div data-section-id="{{ section.id }}" data-section-type="header-section">
-  <header role="banner">
+  <header role="banner" class="">
+    <!-- Mobile slide out menu -->
+    <div id="menu" class="menu-container">
+      <div class="menu-top-section">
+        {{ 'logo2_fiorella_mobile.png' | asset_url | img_tag: 'Mobile Logo'}}
+        <div id="hide-menu-btn" class="menu-close">
+          <button type="button" name="btn">{{ 'icon-hamburger.png' | asset_url | img_tag: 'Burguer Menu'}}</button>
+        </div>
+      </div>
+      <div class="search-container">
+        <form action="/search" method="get" role="search">
+          <label for="Search" class="label-hidden">
+            {{ 'general.search.placeholder' | t }}
+          </label>
+          <input type="search"
+                 name="q"
+                 id="Search"
+                 value="{{ search.terms | escape }}"
+                 placeholder="{{ 'general.search.placeholder' | t }}">
+          <button type="submit" class="btn">
+            {% include 'icon-search' %}
+            <span class="icon-fallback-text">{{ 'general.search.submit' | t }}</span>
+          </button>
+        </form>
+      </div>
+      <div class="mobile-nav">
+        <h2>BOUQUETS</h2>
+        <h2>BOUTONNIERES</h2>
+        <h2>CENTERPIECES</h2>
+        <h2>CEREMONY</h2>
+      </div>
+      <div class="mobile-user-buttons">
+        <h3>LOG IN</h3>
```

Comment:
>For this options list you may create a menu from Shopify Admin, later this menu will available with the object `linklists`. It's like collections example but instead of iterate `collections` object, you should be iterate `linklists.[menu_name_created].links`

Example:

```
<ul>
  {% for link in linklists.menu_user_buttons %}
    <li><a href="{{ link.url }}">{{ link.title }}</a></li>
  {% endfor %}
<ul>
```